shared/utilities/getTicketStatus/test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint max-len: ["error", 2000] */
```

Comment:
The idea of a linter is that you stick to the style rather than turning it off when you violate the style